### DISCUSSION TITLE
GitHub pages help

### DISCUSSION THREAD

**Discussion by Ekiller98** [community/community#67402] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So, I created a GitHub pages account to practice using html and, in my repo, I have different branches. but whenever I go to my site it only displays what I have in the main branch. Is there a way to Display the other branches whenever I go to my site.

**Comment by MaryaBaloch20** [comment]
To display the content of other branches on your GitHub Pages site, **you'll need to create separate repositories for each branch or use a different hosting service that supports this feature.**
By default, GitHub Pages is set up to display the content from the main or master branch only.

**Comment by JustOptimize** [comment]
Github pages doesn't work with different branches, you can put each of your works in the main branch and build a main index to be used as a catalog. If you need help don't esitate to send an email @ mirko303.staff+help@gmail.com and I will assist you step by step helping you build that
